spec/fortnox/api/repositories/examples/find_examples.rb

```diff
@@ -0,0 +1,25 @@
+shared_examples_for '.find' do |id_attribute|
```

Comment:
The id attribute is also available on the repository, we should not ask for it again.